boards/infineon/kit_xmc72_evk/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. zephyr:board:: kit_xmc72_evk
+
+Overview
+********
+
+The XMC7200 evaluation kit enables you to evaluate and develop your applications using the XMC7200D microcontroller(hereafter called “XMC7200D”). XMC7200D is designed for industrial applications. XMC7200D is a true programmable embedded system-on chip, integrating up to two 350-MHz Arm® Cortex®-M7 as the primary application processors, 100-MHz Arm® Cortex®-M0+ that supports low-power operations, up to 8-MB flash and 1-MB SRAM, Gigabit Ethernet, Controller Area Network Flexible Data-Rate (CAN FD), Secure Digital Host Controller (SDHC) supporting SD/SDIO/eMMC interfaces, programmable analog and digital peripherals that allow faster time-to-market. The evaluation board has a M.2 interface connector for interfacing radio modules-based on AIROC™ Wi-Fi & Bluetooth combos, SMIF dual header compatible with Digilent Pmod for interfacing HYPERBUS™ memories, and headers compatible with Arduino for interfacing Arduino shields. In addition, the board features an onboard programmer/debugger(KitProg3), a 512-Mbit QSPI NOR flash, CAN FD transceiver, Gigabit Ethernet PHY transceiver with RJ45 connector interface, a micro-B connector for USB device interface, three user LEDs, one potentiometer, and two push buttons. The board supports operating voltages from 3.3 V to 5.0 V for XMC7200D.
```

Comment:
That's a VERY long line :) 
Please add some breathing room / paragraph breaks and wrap the whole file with 100-char long lines